.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 **/*.rs.bk
+.*.sw*
```

Comment:
Really you should have this in your own `~/.gitconfig` file, but vim is so ubiquitous, I'll let it in. :)